File: eval/fixtures/legacy-directory-backed-aspnet-commerce/src/Legacy/Billing/BillingRefundRepository.cs
namespace Acme.Legacy.Billing;

public class BillingRefundRepository
{
  public RefundAuditResult WriteRefundAuditEntry(string refundId)
  {
    var sql = "EXEC billing.rebuild_aspnet_refund_audit; SELECT * FROM billing.aspnet_refund_audit_entries";
    return new RefundAuditResult(sql);
  }
}

File: eval/fixtures/legacy-directory-backed-jaxws-commerce/src/main/java/com/acme/legacy/billing/BillingRefundRepository.java
package com.acme.legacy.billing;

public class BillingRefundRepository {
  public RefundAuditResult writeRefundAuditEntry(String refundId) {
    String sql = "CALL rebuild_jaxws_refund_audit(); SELECT * FROM billing.jaxws_refund_audit_entries";
    return new RefundAuditResult(sql);
  }
}
